components/org.wso2.carbon.identity.data.publisher.oauth/src/main/java/org/wso2/carbon/identity/data/publisher/oauth/listener/PasswordGrantAuditLogger.java

```diff
@@ -80,6 +82,16 @@ public void onPostTokenIssue(OAuth2AccessTokenReqDTO tokenReqDTO,
             auditResult = FrameworkConstants.AUDIT_FAILED;
         }
 
+        if (LoggerUtils.isLogMaskingEnable) {
+            String maskedUsername = LoggerUtils.getMaskedContent(requestInitiator);
```

Comment:
Do we need to store this as a variable? we can directly assign the value to `requestInitiator` inside the second if blocks [https://github.com/wso2-extensions/identity-data-publisher-oauth/pull/95/files#diff-af77d0fbce54dcd592fb2a53f595d0b64b86bd06af4db5cda4ab0a02c2b6301cR91](https://github.com/wso2-extensions/identity-data-publisher-oauth/pull/95/files#diff-af77d0fbce54dcd592fb2a53f595d0b64b86bd06af4db5cda4ab0a02c2b6301cR91)